pkg/cluster/manager/display.go

```diff
@@ -53,6 +54,23 @@ type InstInfo struct {
 	Port          int
 }
 
+type DashboardInfo struct {
+	ClusterType    string `json:"cluster_type"`
+	ClusterName    string `json:"cluster_name"`
+	ClusterVersion string `json:"cluster_version"`
+	SSHType        string `json:"ssh_type"`
+	TLSEnabled     bool   `json:"tls_enabled"`
+	TLSCACert      string `json:"tls_ca_cert"`
+	TLSClientCert  string `json:"tls_client_cert"`
+	TLSClientKey   string `json:"tls_client_key"`
+	URL            string `json:"url"`
```

Comment:
I think it's better be `dashboard_url`, as it's the URL of our builtin dashboard in PD, we may have other URLs or links to something else. (there is no such thing as a general "URL" for a TiDB cluster)